ngraph/test/type_prop/square_difference.cpp

```diff
@@ -0,0 +1,9 @@
+// Copyright (C) 2018-2021 Intel Corporation
```

Comment:
// Copyright (C) 2021 Intel Corporation

Reply:
I found out that I missed spell the file name, and the type_prop/square_difference.cpp file was already created containing single test. So I changed it  to match the new formula, but now I dont know if I should leave the old date 2018-2021 or still change it to 2021

Reply:
If file existed then the old date.